functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
Will do - I'll add the equivalent `get-` and `set-` forms of the `DbaSharedSchedule` commands also. Should I close this PR for now or leave it and just update it?